### PR TITLE
WIP admin-api-sdk

### PR DESCRIPTION
**Pull request by allouis**
Will split out commits to make it easier to review this is just an MVP using `node-fetch`

### CODE REVIEW

File: packages/admin-api/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# http://editorconfig.org
```
Comment (ErisDS) [inline packages/admin-api/.editorconfig]: Can you please manually delete this file? 

I have already pushed a bug fix - it shouldn't get generated again. Sorry!

Comment (allouis) [reply packages/admin-api/.editorconfig]: sure thing! :relaxed:

File: packages/admin-api/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "repository": "https://github.com/TryGhost/Ghost-SDKs/tree/master/packages/admin-api",
+  "author": "Ghost Foundation",
+  "license": "MIT",
+  "main": "index.js",
+  "scripts": {
+    "dev": "echo \"Implement me!\"",
+    "test": "NODE_ENV=testing mocha ./test/**/*.test.js ./test/*.test.js",
+    "lint": "eslint . --ext .js --cache",
+    "posttest": "yarn lint"
+  },
```
Comment (ErisDS) [inline packages/admin-api/package.json]: Found another slimer bug (again fixed it and updated) this block should be inserted here:

```
 "publishConfig": {
    "access": "public"
  },
```

**Comment by ErisDS** [comment]
Most important thing in terms of review are naming and the api - i.e. all the design bits need to go through a design review.

**Comment by allouis** [comment]
Closing because this isn't under active development

**Pull request closed without merging**